packages/rollup-plugin-minify-html-literals/package.json

```diff
@@ -3,10 +3,117 @@
   "version": "0.0.0",
   "description": "Rollup plugin to minify HTML template literal strings",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "types": "index.d.ts",
   "author": "Google LLC",
   "license": "BSD-3-Clause",
-  "private": true
+  "private": true,
+  "type": "module",
+  "files": [
+    "index.*",
+    "/lib/"
+  ],
+  "keywords": [
+    "rollup-plugin",
+    "minify",
+    "html",
+    "literal",
+    "literals",
+    "template",
+    "tagged",
+    "lit-html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/rollup-plugin-minify-html-literals"
+  },
+  "scripts": {
+    "build": "wireit",
+    "test": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts:types"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "tsconfig.json"
+      ],
+      "output": [
+        "index.{js,js.map,d.ts,d.ts.map}",
+        "lib/**/*.{js,js.map,d.ts,d.ts.map}",
+        "!lib/types/*.d.ts",
+        "test/**/*.{js,js.map,d.ts,d.ts.map}",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror src/lib/types lib/types \"*.d.ts\"",
+      "dependencies": [
+        "../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "lib/types/*.d.ts"
+      ]
+    },
+    "test": {
+      "command": "nyc mocha -r ts-node/register -r source-map-support/register test/*.js",
+      "dependencies": [
+        "build"
+      ],
+      "files": [],
+      "output": [
+        ".nyc_output/"
+      ]
+    }
+  },
+  "nyc": {
+    "extension": [
+      ".ts"
+    ],
+    "include": [
+      "src/**/*.ts"
+    ],
+    "exclude": [
+      "src/**/*.d.ts"
+    ],
+    "reporter": [
+      "text-summary"
+    ],
+    "all": true
+  },
+  "dependencies": {
+    "clean-css": "^4.2.1",
+    "html-minifier": "^4.0.0",
+    "magic-string": "^0.25.0",
+    "rollup-pluginutils": "^2.8.2",
+    "typescript": "^2.9.2 || ^3.0.0 || ^4.0.0 || ^5.0.0"
```

Comment:
We should tighten this up, since none of our other tools support such old versions. What are we testing against?